src/common/lint/add-linters.ts

```diff
@@ -0,0 +1,28 @@
+import type {Linter} from 'eslint'
+import {NodeProject} from 'projen/lib/javascript'
+import {addLintConfigs} from './add-lint-configs'
+import {addLintScripts} from './add-lint-scripts'
+
+export const addLinters = (
+  project: NodeProject,
+  lintPaths: Array<string>,
+  extraEslintConfigs: Array<Linter.Config> = [],
+): void => {
+  project.addDevDeps(
+    '@ottofeller/eslint-plugin-ottofeller@0.1.3',
+    '@typescript-eslint/eslint-plugin@5.58.0',
+    '@typescript-eslint/parser@5.58.0',
+    '@typescript-eslint/typescript-estree@5.58.0',
+    'eslint-plugin-eslint-comments@3.2.0',
+    'eslint-plugin-import@2.27.5',
+    'eslint-plugin-react@7.32.2',
+    'eslint-plugin-react-hooks@4.6.0',
+    'eslint-plugin-tailwindcss@3.11.0',
+    'eslint@8',
+    'prettier@2.8.7',
+    'prettier-plugin-organize-imports@3.2.2',
+  )
+
+  addLintScripts(project, lintPaths)
```

Comment:
@quesabe if these functions are not used separately from `addLinters` then maybe we could inline them here?

Reply:
Sure. I thought to leave at for later, but we certainly can do it. Let's do it here.